### PR TITLE
:sparkles: (dashboards) ability to quick-edit widget names from inner report pages

### PR DESCRIPTION
**Pull request by MatissJanis**
https://github.com/actualbudget/actual/issues/3282

### CODE REVIEW

File: packages/desktop-client/src/components/reports/Header.tsx [hunk 1]
```diff
@@ -73,141 +70,140 @@ export function Header({
         flexShrink: 0,
       }}
     >
-      {!['/reports/custom'].includes(path) && (
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/reports/Header.tsx:76]: This component is not used in the custom reports page anymore.

File: packages/desktop-client/src/components/reports/Header.tsx [hunk 2]
```diff
@@ -73,141 +70,140 @@ export function Header({
...
-                onApply={onApply}
-                hover={false}
-                exclude={undefined}
-              />
-            )}
+            {children}
           </View>
+        ) : (
+          <View style={{ flex: 1 }} />
+        )}
```
[the remark this answers is not in the document]
Comment (MatissJanis) [reply packages/desktop-client/src/components/reports/Header.tsx:204]: Ok good feedback, but out of scope for this PR (I'm really only changing the indentation in the above lines, not the underlying logic).

File: packages/desktop-client/src/components/reports/Header.tsx [hunk 3]
```diff
@@ -73,141 +70,140 @@ export function Header({
...
+            flexWrap: 'wrap',
+          }}
+        >
+          {show1Month && (
+            <Button
+              variant="bare"
+              onPress={() => onChangeDates(...getLatestRange(1))}
             >
-              {mode === 'static' ? 'Static' : 'Live'}
+              1 month
```
[the remark this answers is not in the document]
Comment (MatissJanis) [reply packages/desktop-client/src/components/reports/Header.tsx:150]: Same as above: good comment, but out of scope for this PR.

File: packages/desktop-client/src/components/reports/Header.tsx [hunk 4]
```diff
@@ -73,141 +70,140 @@ export function Header({
...
+              const [newStart, newEnd] = calculateTimeRange({
+                start,
+                end,
+                mode: newMode,
+              });
+
+              onChangeDates(newStart, newEnd, newMode);
+            }}
+          >
+            {mode === 'static' ? 'Static' : 'Live'}
```
[the remark this answers is not in the document]
Comment (MatissJanis) [reply packages/desktop-client/src/components/reports/Header.tsx:95]: Same as above: good comment, but out of scope for this PR.

**Review by youngcw — APPROVED** [review]
Looks good

**Pull request merged**